src/stactools/sentinel2/constants.py

```diff
@@ -31,79 +31,79 @@
     'B01':
     Band.create(name='B01',
                 common_name='coastal',
-                description='Band 1 - Coastal aerosol',
+                description='Coastal aerosol (band 1)',
                 center_wavelength=0.443,
                 full_width_half_max=0.027),
     'B02':
     Band.create(name='B02',
                 common_name='blue',
-                description='Band 2 - Blue',
+                description='Blue (band 2)',
                 center_wavelength=0.490,
                 full_width_half_max=0.098),
     'B03':
     Band.create(name='B03',
                 common_name='green',
-                description='Band 3 - Green',
+                description='Green (band 3)',
                 center_wavelength=0.560,
                 full_width_half_max=0.045),
     'B04':
     Band.create(name='B04',
                 common_name='red',
-                description='Band 4 - Red',
+                description='Red (band 3)',
                 center_wavelength=0.665,
                 full_width_half_max=0.038),
     'B05':
     Band.create(name='B05',
                 common_name='rededge',
-                description='Band 5 - Vegetation red edge 1',
+                description='Red edge 1 (band 5)',
                 center_wavelength=0.704,
                 full_width_half_max=0.019),
     'B06':
     Band.create(name='B06',
                 common_name='rededge',
-                description='Band 6 - Vegetation red edge 2',
+                description='Red edge 2 (band 6)',
                 center_wavelength=0.740,
                 full_width_half_max=0.018),
     'B07':
     Band.create(name='B07',
                 common_name='rededge',
-                description='Band 7 - Vegetation red edge 3',
+                description='Red edge 3 (band 7)',
                 center_wavelength=0.783,
                 full_width_half_max=0.028),
     'B08':
     Band.create(name='B08',
                 common_name='nir',
-                description='Band 8 - NIR',
+                description='NIR 1 (band 8)',
                 center_wavelength=0.842,
                 full_width_half_max=0.145),
     'B8A':
     Band.create(name='B8A',
                 common_name='nir08',
-                description='Band 8A - Vegetation red edge 4',
+                description='NIR 2 (band 8A)',
                 center_wavelength=0.865,
                 full_width_half_max=0.033),
     'B09':
     Band.create(name='B09',
                 common_name='nir09',
-                description='Band 9 - Water vapor',
```

Comment:
changing this to NIR as to not confuse it with the WVP product -- it used for water vapor, but it's not water vapor